tests/spec/fs.readdir.spec.js

```diff
@@ -54,4 +54,33 @@ describe('fs.readdir', function() {
       });
     });
   });
+
+  it('(promise) should be a function', function() {
```

Comment:
Just to nitpick here, you can use () => {...} to declare a function instead of function() {...}. It's a shorter syntax so less typing.